### ISSUE TITLE
Accessibility: Notification button is missing aria label

### ISSUE THREAD

**Issue by capeoples** [Esri/global-nav#233]
Per an axe accessibility report: 

![Screen Shot 2019-05-08 at 10 37 07 AM](https://user-images.githubusercontent.com/30473086/57395572-44c32080-717d-11e9-9865-44a46348bd0d.png)

**Comment by paulcpederson** [comment]
Should be fixed in [📦  v1.4.0](https://github.com/Esri/global-nav/releases/tag/v1.4.0)
